news/2018-05-16-performance-updates.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "New Scoring Updates: osu! and osu!mania"
+date: 2018-05-16 14:00:00 +0000
+---
+
+A brand new set of pp adjustments have made their way through the pipeline for osu! and osu!mania – the first of their kind in several years! Wonder how the changes might have affected you and your ranks? Read on!
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+Over the past few weeks, much forward progress has been made on the [performance processor](https://github.com/ppy/osu-performance) (mainly thanks to [Tom94](https://github.com/Tom94)). As a result, we have also been able to quickly iterate over community proposals and begin breathing life back into the performance and star rating algorithms to keep them relevant.
+
+**Note that as these changes are applied, ranks may fly around a bit.** We are doing our best to alleviate the visual effect of reprocessing, but as this is the first time we've done this kind of operation in a while it is a learning experience – there's a lot of data to process!
+
+We begin this journey with the following changes:
+
+## inactive players
+
+Until recently, we have not been marking inactive players (no score recorded in 3 months) as such in all cases, which means that as we perform this reprocessing, **all users will generally see a sudden increase in rank**.
+
+## osu!
+
+### [Adjusting Hidden Mod As Speed Multiplier](https://github.com/ppy/osu-performance/pull/42)
+
+The aim value scales significantly less than it did before but speedValue has now been given a scaling bonus. Simply put, this means that when using Hidden jumps reward less pp and streams reward more pp than before.
+
+### [AR Fix for aim on HD](https://github.com/ppy/osu-performance/pull/47) to fix further [issues raised by NoMoreDiv](https://github.com/ppy/osu-performance/issues/45).
+
+Hidden's aim value now scales slightly less than previously at very high approach rates where using the mod often makes it easier to read and quickly react to. The new calculation now gives a higher scaling to lower approach rates than before as well.
+
```

Comment:
maybe add a "How do these changes affect me?" section containing Toy's gist stuff, aka:

```
- The aim bonus the Hidden mod is reduced. Maps that features more **jumps** or higher jump spacing have been nerfed. 
- This aim reduction is linear based on the approach rate of the map. **Higher AR means a smaller aim bonus than lower AR for the same pattern.**
- The speed bonus for the Hidden mod has been increased. Streams are calculated under the speed category, and now more **spaced streams have been buffed slightly**.
- This is **NOT** an absolute nerf to Hidden as a whole, it is the beginning of a rework in how the mod affects the formula as a whole.
- Playing a map with Hidden will still award more points than not using it with the same accuracy.

Examples:
Cookiezi’s Remote Control play was nerfed from 817 -> 776. Aim maps are nerfed. 
Cookiezi’s Freedom Dive play was buffed from 800 -> 820. Spaced streams are buffed.

```

most of this stuff will just roll off people's heads as it currently is unless they can relate it to something